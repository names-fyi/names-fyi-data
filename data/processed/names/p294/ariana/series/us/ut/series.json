[{"y":1990,"t":6,"m":0,"f":6,"mr":null,"fr":410,"mp":0,"fp":0.00044277175116227583},{"y":1991,"t":5,"m":0,"f":5,"mr":null,"fr":470,"mp":0,"fp":0.0003762510346903454},{"y":1992,"t":8,"m":0,"f":8,"mr":null,"fr":329,"mp":0,"fp":0.0005794582065768506},{"y":1993,"t":14,"m":0,"f":14,"mr":null,"fr":204,"mp":0,"fp":0.0010349671028313742},{"y":1994,"t":15,"m":0,"f":15,"mr":null,"fr":200,"mp":0,"fp":0.0010787486515641855},{"y":1995,"t":15,"m":0,"f":15,"mr":null,"fr":217,"mp":0,"fp":0.0010135135135135136},{"y":1996,"t":17,"m":0,"f":17,"mr":null,"fr":205,"mp":0,"fp":0.0011165112307894391},{"y":1997,"t":18,"m":0,"f":18,"mr":null,"fr":203,"mp":0,"fp":0.0011551049220304178},{"y":1998,"t":20,"m":0,"f":20,"mr":null,"fr":190,"mp":0,"fp":0.0012108736453351092},{"y":1999,"t":24,"m":0,"f":24,"mr":null,"fr":164,"mp":0,"fp":0.0014452607491268216},{"y":2000,"t":24,"m":0,"f":24,"mr":null,"fr":180,"mp":0,"fp":0.001408368053517986},{"y":2001,"t":27,"m":0,"f":27,"mr":null,"fr":162,"mp":0,"fp":0.001557003633008477},{"y":2002,"t":46,"m":0,"f":46,"mr":null,"fr":93,"mp":0,"fp":0.002614527679890872},{"y":2003,"t":33,"m":0,"f":33,"mr":null,"fr":139,"mp":0,"fp":0.001827343706739022},{"y":2004,"t":37,"m":0,"f":37,"mr":null,"fr":124,"mp":0,"fp":0.002026397940741552},{"y":2005,"t":31,"m":0,"f":31,"mr":null,"fr":149,"mp":0,"fp":0.0016850573463064631},{"y":2006,"t":25,"m":0,"f":25,"mr":null,"fr":190,"mp":0,"fp":0.001325275657336726},{"y":2007,"t":37,"m":0,"f":37,"mr":null,"fr":127,"mp":0,"fp":0.0018865038494875848},{"y":2008,"t":39,"m":0,"f":39,"mr":null,"fr":126,"mp":0,"fp":0.0019729852784944606},{"y":2009,"t":40,"m":0,"f":40,"mr":null,"fr":117,"mp":0,"fp":0.0021015025743406535},{"y":2010,"t":31,"m":0,"f":31,"mr":null,"fr":145,"mp":0,"fp":0.0017078007933010136},{"y":2011,"t":29,"m":0,"f":29,"mr":null,"fr":163,"mp":0,"fp":0.0016157789168709605},{"y":2012,"t":32,"m":0,"f":32,"mr":null,"fr":136,"mp":0,"fp":0.001789909385837342},{"y":2013,"t":44,"m":0,"f":44,"mr":null,"fr":94,"mp":0,"fp":0.0024929178470254956},{"y":2014,"t":32,"m":0,"f":32,"mr":null,"fr":134,"mp":0,"fp":0.0017686398054496213},{"y":2015,"t":40,"m":0,"f":40,"mr":null,"fr":105,"mp":0,"fp":0.00224782242202866},{"y":2016,"t":40,"m":0,"f":40,"mr":null,"fr":110,"mp":0,"fp":0.0022182786157941437},{"y":2017,"t":30,"m":0,"f":30,"mr":null,"fr":138,"mp":0,"fp":0.0017623215649415498},{"y":2018,"t":25,"m":0,"f":25,"mr":null,"fr":167,"mp":0,"fp":0.001499970000599988},{"y":2019,"t":28,"m":0,"f":28,"mr":null,"fr":147,"mp":0,"fp":0.0016697477488222315},{"y":2020,"t":25,"m":0,"f":25,"mr":null,"fr":160,"mp":0,"fp":0.001555790652809758},{"y":2021,"t":24,"m":0,"f":24,"mr":null,"fr":174,"mp":0,"fp":0.0014220536825265153},{"y":2022,"t":28,"m":0,"f":28,"mr":null,"fr":146,"mp":0,"fp":0.0017510944340212633}]