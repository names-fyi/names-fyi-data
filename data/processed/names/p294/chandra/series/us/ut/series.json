[{"y":1969,"t":6,"m":0,"f":6,"mr":null,"fr":316,"mp":0,"fp":0.0006387735547748323},{"y":1972,"t":5,"m":0,"f":5,"mr":null,"fr":370,"mp":0,"fp":0.00047460844803037496},{"y":1973,"t":5,"m":0,"f":5,"mr":null,"fr":383,"mp":0,"fp":0.00046070211001566385},{"y":1974,"t":7,"m":0,"f":7,"mr":null,"fr":296,"mp":0,"fp":0.0005974225484338994},{"y":1976,"t":7,"m":0,"f":7,"mr":null,"fr":331,"mp":0,"fp":0.0005064023728568328},{"y":1977,"t":10,"m":0,"f":10,"mr":null,"fr":267,"mp":0,"fp":0.0006803646754660498},{"y":1978,"t":9,"m":0,"f":9,"mr":null,"fr":309,"mp":0,"fp":0.0005934326783594883},{"y":1980,"t":8,"m":0,"f":8,"mr":null,"fr":347,"mp":0,"fp":0.0004860562610122122},{"y":1982,"t":5,"m":0,"f":5,"mr":null,"fr":479,"mp":0,"fp":0.00030485945978903724},{"y":1983,"t":6,"m":0,"f":6,"mr":null,"fr":409,"mp":0,"fp":0.0003892312682452157},{"y":1984,"t":6,"m":0,"f":6,"mr":null,"fr":418,"mp":0,"fp":0.0004018215912135012},{"y":1985,"t":6,"m":0,"f":6,"mr":null,"fr":404,"mp":0,"fp":0.0004182933630786392},{"y":1986,"t":7,"m":0,"f":7,"mr":null,"fr":363,"mp":0,"fp":0.0005011454753722795},{"y":1987,"t":12,"m":0,"f":12,"mr":null,"fr":225,"mp":0,"fp":0.0008957897879964168},{"y":1989,"t":6,"m":0,"f":6,"mr":null,"fr":416,"mp":0,"fp":0.0004463953574882821},{"y":1990,"t":7,"m":0,"f":7,"mr":null,"fr":360,"mp":0,"fp":0.0005165670430226551},{"y":1991,"t":6,"m":0,"f":6,"mr":null,"fr":420,"mp":0,"fp":0.0004515012416284145},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":496,"mp":0,"fp":0.00036216137911053165},{"y":1993,"t":6,"m":0,"f":6,"mr":null,"fr":426,"mp":0,"fp":0.00044355732978487467}]