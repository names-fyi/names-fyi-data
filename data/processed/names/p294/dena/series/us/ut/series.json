[{"y":1959,"t":7,"m":0,"f":7,"mr":null,"fr":276,"mp":0,"fp":0.0006791500921703696},{"y":1961,"t":5,"m":0,"f":5,"mr":null,"fr":354,"mp":0,"fp":0.00047192071731949034},{"y":1962,"t":8,"m":0,"f":8,"mr":null,"fr":270,"mp":0,"fp":0.0007580782715815408},{"y":1963,"t":11,"m":0,"f":11,"mr":null,"fr":211,"mp":0,"fp":0.0011076427348706071},{"y":1964,"t":7,"m":0,"f":7,"mr":null,"fr":284,"mp":0,"fp":0.000744047619047619},{"y":1967,"t":10,"m":0,"f":10,"mr":null,"fr":203,"mp":0,"fp":0.0011173184357541898},{"y":1968,"t":6,"m":0,"f":6,"mr":null,"fr":298,"mp":0,"fp":0.000664451827242525},{"y":1969,"t":11,"m":0,"f":11,"mr":null,"fr":205,"mp":0,"fp":0.0011710848504205259},{"y":1970,"t":11,"m":0,"f":11,"mr":null,"fr":217,"mp":0,"fp":0.0010466222645099904},{"y":1971,"t":7,"m":0,"f":7,"mr":null,"fr":304,"mp":0,"fp":0.0006488089721012142},{"y":1972,"t":11,"m":0,"f":11,"mr":null,"fr":215,"mp":0,"fp":0.001044138585666825},{"y":1973,"t":6,"m":0,"f":6,"mr":null,"fr":343,"mp":0,"fp":0.0005528425320187966},{"y":1974,"t":7,"m":0,"f":7,"mr":null,"fr":300,"mp":0,"fp":0.0005974225484338994},{"y":1975,"t":5,"m":0,"f":5,"mr":null,"fr":410,"mp":0,"fp":0.00040489108429832374},{"y":1978,"t":5,"m":0,"f":5,"mr":null,"fr":468,"mp":0,"fp":0.00032968482131082687},{"y":1984,"t":5,"m":0,"f":5,"mr":null,"fr":481,"mp":0,"fp":0.000334851326011251}]