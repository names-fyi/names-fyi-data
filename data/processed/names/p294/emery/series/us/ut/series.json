[{"y":1915,"t":6,"m":6,"f":0,"mr":147,"fr":null,"mp":0.0016357688113413304,"fp":0},{"y":1922,"t":6,"m":6,"f":0,"mr":168,"fr":null,"mp":0.0013015184381778742,"fp":0},{"y":2000,"t":7,"m":0,"f":7,"mr":null,"fr":501,"mp":0,"fp":0.0004107740156094126},{"y":2001,"t":9,"m":0,"f":9,"mr":null,"fr":422,"mp":0,"fp":0.0005190012110028257},{"y":2002,"t":10,"m":0,"f":10,"mr":null,"fr":384,"mp":0,"fp":0.0005683755825849721},{"y":2003,"t":13,"m":0,"f":13,"mr":null,"fr":330,"mp":0,"fp":0.000719862672351736},{"y":2004,"t":15,"m":0,"f":15,"mr":null,"fr":299,"mp":0,"fp":0.0008215126786790076},{"y":2005,"t":18,"m":0,"f":18,"mr":null,"fr":266,"mp":0,"fp":0.000978420394629559},{"y":2006,"t":18,"m":0,"f":18,"mr":null,"fr":248,"mp":0,"fp":0.0009541984732824427},{"y":2007,"t":33,"m":0,"f":33,"mr":null,"fr":147,"mp":0,"fp":0.0016825574873808188},{"y":2008,"t":36,"m":5,"f":31,"mr":710,"fr":153,"mp":0.00021095266222259724,"fp":0.0015682703495725199},{"y":2009,"t":33,"m":0,"f":33,"mr":null,"fr":145,"mp":0,"fp":0.0017337396238310391},{"y":2010,"t":48,"m":5,"f":43,"mr":677,"fr":102,"mp":0.00022665457842248413,"fp":0.0023688849713530188},{"y":2011,"t":61,"m":8,"f":53,"mr":496,"fr":74,"mp":0.00037407649864397267,"fp":0.0029529752618676175},{"y":2012,"t":70,"m":7,"f":63,"mr":569,"fr":51,"mp":0.00032335550628233555,"fp":0.003523884103367267},{"y":2013,"t":67,"m":0,"f":67,"mr":null,"fr":53,"mp":0,"fp":0.0037960339943342775},{"y":2014,"t":64,"m":0,"f":64,"mr":null,"fr":54,"mp":0,"fp":0.0035372796108992426},{"y":2015,"t":83,"m":0,"f":83,"mr":null,"fr":40,"mp":0,"fp":0.004664231525709469},{"y":2016,"t":84,"m":5,"f":79,"mr":703,"fr":43,"mp":0.00023862931322483655,"fp":0.004381100266193434},{"y":2017,"t":64,"m":0,"f":64,"mr":null,"fr":53,"mp":0,"fp":0.0037596193385419727},{"y":2018,"t":82,"m":0,"f":82,"mr":null,"fr":38,"mp":0,"fp":0.004919901601967961},{"y":2019,"t":96,"m":6,"f":90,"mr":603,"fr":27,"mp":0.00031043046357615895,"fp":0.00536704633550003},{"y":2020,"t":85,"m":6,"f":79,"mr":621,"fr":35,"mp":0.0003117692907248636,"fp":0.0049162984628788355},{"y":2021,"t":74,"m":0,"f":74,"mr":null,"fr":42,"mp":0,"fp":0.0043846655211234225},{"y":2022,"t":60,"m":0,"f":60,"mr":null,"fr":54,"mp":0,"fp":0.00375234521575985}]