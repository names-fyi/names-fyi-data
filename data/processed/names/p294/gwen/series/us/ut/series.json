[{"y":1912,"t":6,"m":0,"f":6,"mr":null,"fr":102,"mp":0,"fp":0.0033112582781456954},{"y":1915,"t":11,"m":0,"f":11,"mr":null,"fr":111,"mp":0,"fp":0.002728174603174603},{"y":1916,"t":9,"m":0,"f":9,"mr":null,"fr":128,"mp":0,"fp":0.002019295490240072},{"y":1917,"t":12,"m":0,"f":12,"mr":null,"fr":113,"mp":0,"fp":0.0025850926324859974},{"y":1918,"t":10,"m":0,"f":10,"mr":null,"fr":146,"mp":0,"fp":0.001904036557501904},{"y":1919,"t":5,"m":0,"f":5,"mr":null,"fr":216,"mp":0,"fp":0.00106315118009781},{"y":1920,"t":8,"m":0,"f":8,"mr":null,"fr":159,"mp":0,"fp":0.0015717092337917485},{"y":1921,"t":7,"m":0,"f":7,"mr":null,"fr":170,"mp":0,"fp":0.001366120218579235},{"y":1922,"t":11,"m":0,"f":11,"mr":null,"fr":125,"mp":0,"fp":0.0022137250955926746},{"y":1923,"t":13,"m":0,"f":13,"mr":null,"fr":108,"mp":0,"fp":0.0025630914826498424},{"y":1924,"t":10,"m":0,"f":10,"mr":null,"fr":122,"mp":0,"fp":0.0019813750743015654},{"y":1925,"t":13,"m":0,"f":13,"mr":null,"fr":107,"mp":0,"fp":0.002615167974250654},{"y":1926,"t":11,"m":0,"f":11,"mr":null,"fr":118,"mp":0,"fp":0.0022793203481143802},{"y":1927,"t":10,"m":0,"f":10,"mr":null,"fr":119,"mp":0,"fp":0.0021358393848782574},{"y":1928,"t":11,"m":0,"f":11,"mr":null,"fr":117,"mp":0,"fp":0.0024091108190976785},{"y":1929,"t":12,"m":0,"f":12,"mr":null,"fr":108,"mp":0,"fp":0.0026472534745201853},{"y":1930,"t":17,"m":0,"f":17,"mr":null,"fr":75,"mp":0,"fp":0.0035549979088247597},{"y":1931,"t":14,"m":0,"f":14,"mr":null,"fr":82,"mp":0,"fp":0.003217651114686279},{"y":1932,"t":13,"m":0,"f":13,"mr":null,"fr":80,"mp":0,"fp":0.0030176415970287838},{"y":1933,"t":14,"m":0,"f":14,"mr":null,"fr":81,"mp":0,"fp":0.0031832651205093224},{"y":1934,"t":12,"m":0,"f":12,"mr":null,"fr":99,"mp":0,"fp":0.002722940776038121},{"y":1935,"t":11,"m":0,"f":11,"mr":null,"fr":110,"mp":0,"fp":0.0023655913978494624},{"y":1936,"t":13,"m":0,"f":13,"mr":null,"fr":87,"mp":0,"fp":0.0028515025224830006},{"y":1937,"t":12,"m":0,"f":12,"mr":null,"fr":103,"mp":0,"fp":0.0026472534745201853},{"y":1938,"t":12,"m":0,"f":12,"mr":null,"fr":101,"mp":0,"fp":0.002509410288582183},{"y":1939,"t":14,"m":0,"f":14,"mr":null,"fr":88,"mp":0,"fp":0.0029424127784783522},{"y":1940,"t":6,"m":0,"f":6,"mr":null,"fr":180,"mp":0,"fp":0.0011985617259288853},{"y":1941,"t":8,"m":0,"f":8,"mr":null,"fr":141,"mp":0,"fp":0.0015791551519936833},{"y":1942,"t":9,"m":0,"f":9,"mr":null,"fr":137,"mp":0,"fp":0.001495016611295681},{"y":1943,"t":9,"m":0,"f":9,"mr":null,"fr":154,"mp":0,"fp":0.0013049151805132667},{"y":1944,"t":9,"m":0,"f":9,"mr":null,"fr":156,"mp":0,"fp":0.0013632232656770676},{"y":1945,"t":7,"m":0,"f":7,"mr":null,"fr":182,"mp":0,"fp":0.0011248593925759281},{"y":1946,"t":10,"m":0,"f":10,"mr":null,"fr":152,"mp":0,"fp":0.0013764624913971094},{"y":1947,"t":8,"m":0,"f":8,"mr":null,"fr":200,"mp":0,"fp":0.0009251763617439575},{"y":1948,"t":8,"m":0,"f":8,"mr":null,"fr":197,"mp":0,"fp":0.0009642039291310112},{"y":1949,"t":10,"m":0,"f":10,"mr":null,"fr":175,"mp":0,"fp":0.0012155099064057372},{"y":1950,"t":16,"m":0,"f":16,"mr":null,"fr":123,"mp":0,"fp":0.0018781547129944828},{"y":1951,"t":12,"m":0,"f":12,"mr":null,"fr":173,"mp":0,"fp":0.0012996859092386007},{"y":1952,"t":7,"m":0,"f":7,"mr":null,"fr":259,"mp":0,"fp":0.0007305364224587769},{"y":1953,"t":7,"m":0,"f":7,"mr":null,"fr":259,"mp":0,"fp":0.0007131214343928281},{"y":1954,"t":8,"m":0,"f":8,"mr":null,"fr":237,"mp":0,"fp":0.0008012820512820513},{"y":1955,"t":9,"m":0,"f":9,"mr":null,"fr":227,"mp":0,"fp":0.0008796794057276903},{"y":1956,"t":10,"m":0,"f":10,"mr":null,"fr":232,"mp":0,"fp":0.000948676596148373},{"y":1957,"t":10,"m":0,"f":10,"mr":null,"fr":219,"mp":0,"fp":0.0009504799923961601},{"y":1958,"t":7,"m":0,"f":7,"mr":null,"fr":281,"mp":0,"fp":0.0006782945736434108},{"y":1959,"t":6,"m":0,"f":6,"mr":null,"fr":313,"mp":0,"fp":0.0005821286504317454},{"y":1960,"t":7,"m":0,"f":7,"mr":null,"fr":299,"mp":0,"fp":0.000655983506700403},{"y":1961,"t":8,"m":0,"f":8,"mr":null,"fr":279,"mp":0,"fp":0.0007550731477111845},{"y":1962,"t":5,"m":0,"f":5,"mr":null,"fr":367,"mp":0,"fp":0.000473798919738463},{"y":1963,"t":16,"m":0,"f":16,"mr":null,"fr":158,"mp":0,"fp":0.0016111167052663376},{"y":1965,"t":7,"m":0,"f":7,"mr":null,"fr":272,"mp":0,"fp":0.0008110300081103001},{"y":1970,"t":5,"m":0,"f":5,"mr":null,"fr":370,"mp":0,"fp":0.0004757373929590866},{"y":1976,"t":6,"m":0,"f":6,"mr":null,"fr":376,"mp":0,"fp":0.00043405917673442813},{"y":1978,"t":5,"m":0,"f":5,"mr":null,"fr":471,"mp":0,"fp":0.00032968482131082687},{"y":2004,"t":5,"m":0,"f":5,"mr":null,"fr":743,"mp":0,"fp":0.0002738375595596692},{"y":2005,"t":5,"m":0,"f":5,"mr":null,"fr":734,"mp":0,"fp":0.00027178344295265535},{"y":2006,"t":11,"m":0,"f":11,"mr":null,"fr":406,"mp":0,"fp":0.0005831212892281595},{"y":2007,"t":10,"m":0,"f":10,"mr":null,"fr":451,"mp":0,"fp":0.0005098659052669148},{"y":2008,"t":6,"m":0,"f":6,"mr":null,"fr":696,"mp":0,"fp":0.00030353619669145547},{"y":2009,"t":17,"m":0,"f":17,"mr":null,"fr":285,"mp":0,"fp":0.0008931385940947777},{"y":2010,"t":7,"m":0,"f":7,"mr":null,"fr":589,"mp":0,"fp":0.0003856324371970031},{"y":2011,"t":9,"m":0,"f":9,"mr":null,"fr":475,"mp":0,"fp":0.0005014486293737464},{"y":2012,"t":21,"m":0,"f":21,"mr":null,"fr":221,"mp":0,"fp":0.0011746280344557558},{"y":2013,"t":22,"m":0,"f":22,"mr":null,"fr":209,"mp":0,"fp":0.0012464589235127478},{"y":2014,"t":24,"m":0,"f":24,"mr":null,"fr":200,"mp":0,"fp":0.001326479854087216},{"y":2015,"t":22,"m":0,"f":22,"mr":null,"fr":210,"mp":0,"fp":0.001236302332115763},{"y":2016,"t":29,"m":0,"f":29,"mr":null,"fr":161,"mp":0,"fp":0.0016082519964507542},{"y":2017,"t":20,"m":0,"f":20,"mr":null,"fr":218,"mp":0,"fp":0.0011748810432943664},{"y":2018,"t":18,"m":0,"f":18,"mr":null,"fr":244,"mp":0,"fp":0.0010799784004319913},{"y":2019,"t":26,"m":0,"f":26,"mr":null,"fr":163,"mp":0,"fp":0.0015504800524777864},{"y":2020,"t":28,"m":0,"f":28,"mr":null,"fr":143,"mp":0,"fp":0.001742485531146929},{"y":2021,"t":17,"m":0,"f":17,"mr":null,"fr":254,"mp":0,"fp":0.0010072880251229485},{"y":2022,"t":28,"m":0,"f":28,"mr":null,"fr":148,"mp":0,"fp":0.0017510944340212633}]