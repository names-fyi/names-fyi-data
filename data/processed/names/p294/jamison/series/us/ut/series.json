[{"y":1970,"t":6,"m":6,"f":0,"mr":269,"fr":null,"mp":0.00048638132295719845,"fp":0},{"y":1973,"t":5,"m":5,"f":0,"mr":305,"fr":null,"mp":0.00039234149403640927,"fp":0},{"y":1974,"t":5,"m":5,"f":0,"mr":336,"fr":null,"mp":0.00036589828027808267,"fp":0},{"y":1975,"t":5,"m":5,"f":0,"mr":338,"fr":null,"mp":0.00034578146611341634,"fp":0},{"y":1981,"t":7,"m":7,"f":0,"mr":310,"fr":null,"mp":0.0003680917074196771,"fp":0},{"y":1983,"t":6,"m":6,"f":0,"mr":330,"fr":null,"mp":0.00032927230819888045,"fp":0},{"y":1986,"t":6,"m":6,"f":0,"mr":317,"fr":null,"mp":0.0003631961259079903,"fp":0},{"y":1989,"t":6,"m":6,"f":0,"mr":341,"fr":null,"mp":0.0003622313450857281,"fp":0},{"y":1992,"t":10,"m":10,"f":0,"mr":268,"fr":null,"mp":0.0005993048064245475,"fp":0},{"y":1993,"t":7,"m":7,"f":0,"mr":341,"fr":null,"mp":0.00041476565740356696,"fp":0},{"y":1994,"t":8,"m":8,"f":0,"mr":324,"fr":null,"mp":0.0004595852243350376,"fp":0},{"y":1995,"t":5,"m":5,"f":0,"mr":454,"fr":null,"mp":0.00028516026006615715,"fp":0},{"y":1996,"t":7,"m":7,"f":0,"mr":395,"fr":null,"mp":0.00036964672334583094,"fp":0},{"y":1997,"t":5,"m":5,"f":0,"mr":525,"fr":null,"mp":0.00026147892479866125,"fp":0},{"y":1998,"t":9,"m":9,"f":0,"mr":357,"fr":null,"mp":0.0004570383912248629,"fp":0},{"y":1999,"t":12,"m":12,"f":0,"mr":307,"fr":null,"mp":0.0005823546539842764,"fp":0},{"y":2000,"t":9,"m":9,"f":0,"mr":381,"fr":null,"mp":0.0004297378599054577,"fp":0},{"y":2001,"t":9,"m":9,"f":0,"mr":389,"fr":null,"mp":0.00042448825582492216,"fp":0},{"y":2002,"t":7,"m":7,"f":0,"mr":472,"fr":null,"mp":0.00032606670393143286,"fp":0},{"y":2003,"t":9,"m":9,"f":0,"mr":411,"fr":null,"mp":0.00041553164966064913,"fp":0},{"y":2004,"t":10,"m":10,"f":0,"mr":386,"fr":null,"mp":0.0004546074464699732,"fp":0},{"y":2005,"t":8,"m":8,"f":0,"mr":463,"fr":null,"mp":0.0003575259206292456,"fp":0},{"y":2006,"t":9,"m":9,"f":0,"mr":444,"fr":null,"mp":0.0003865979381443299,"fp":0},{"y":2007,"t":8,"m":8,"f":0,"mr":518,"fr":null,"mp":0.0003362757461118117,"fp":0},{"y":2008,"t":17,"m":17,"f":0,"mr":309,"fr":null,"mp":0.0007172390515568307,"fp":0},{"y":2009,"t":11,"m":11,"f":0,"mr":420,"fr":null,"mp":0.00047882296609062813,"fp":0},{"y":2010,"t":7,"m":7,"f":0,"mr":557,"fr":null,"mp":0.00031731640979147777,"fp":0},{"y":2011,"t":13,"m":13,"f":0,"mr":357,"fr":null,"mp":0.0006078743102964556,"fp":0},{"y":2012,"t":12,"m":12,"f":0,"mr":386,"fr":null,"mp":0.0005543237250554324,"fp":0},{"y":2013,"t":11,"m":11,"f":0,"mr":418,"fr":null,"mp":0.0005098021040923205,"fp":0},{"y":2014,"t":10,"m":10,"f":0,"mr":447,"fr":null,"mp":0.000464921660700172,"fp":0},{"y":2015,"t":16,"m":16,"f":0,"mr":327,"fr":null,"mp":0.0007538280329799764,"fp":0},{"y":2016,"t":12,"m":12,"f":0,"mr":400,"fr":null,"mp":0.0005727103517396077,"fp":0},{"y":2017,"t":17,"m":17,"f":0,"mr":309,"fr":null,"mp":0.0008373559255245789,"fp":0},{"y":2018,"t":9,"m":9,"f":0,"mr":469,"fr":null,"mp":0.0004621546677621444,"fp":0},{"y":2019,"t":13,"m":13,"f":0,"mr":361,"fr":null,"mp":0.0006725993377483443,"fp":0},{"y":2020,"t":17,"m":17,"f":0,"mr":309,"fr":null,"mp":0.0008833463237204468,"fp":0},{"y":2021,"t":5,"m":5,"f":0,"mr":711,"fr":null,"mp":0.00026123301985370953,"fp":0}]