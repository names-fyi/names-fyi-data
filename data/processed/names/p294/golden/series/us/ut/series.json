[{"y":1913,"t":5,"m":5,"f":0,"mr":125,"fr":null,"mp":0.00249500998003992,"fp":0},{"y":1916,"t":6,"m":6,"f":0,"mr":173,"fr":null,"mp":0.0015166835187057635,"fp":0},{"y":1920,"t":6,"m":6,"f":0,"mr":177,"fr":null,"mp":0.001282051282051282,"fp":0},{"y":1925,"t":5,"m":5,"f":0,"mr":197,"fr":null,"mp":0.0009873617693522906,"fp":0},{"y":1929,"t":5,"m":5,"f":0,"mr":192,"fr":null,"mp":0.0010782833728703904,"fp":0}]